patches/chrome-android-java-src-org-chromium-chrome-browser-customtabs-content-CustomTabActivityTabFactory.java.patch

```diff
@@ -0,0 +1,31 @@
+diff --git a/chrome/android/java/src/org/chromium/chrome/browser/customtabs/content/CustomTabActivityTabFactory.java b/chrome/android/java/src/org/chromium/chrome/browser/customtabs/content/CustomTabActivityTabFactory.java
+index 157946aad58b6efcba212da92effb3dc9e7b30d2..db7638335f72755418c6437894742836b73ca219 100644
+--- a/chrome/android/java/src/org/chromium/chrome/browser/customtabs/content/CustomTabActivityTabFactory.java
++++ b/chrome/android/java/src/org/chromium/chrome/browser/customtabs/content/CustomTabActivityTabFactory.java
+@@ -32,7 +32,7 @@ import javax.inject.Inject;
+ import dagger.Lazy;
+ 
+ /**
+- * Creates {@link Tab}, {@link TabModelSelector}, and {@link ChromeTabCreator}s in the context of a
++ * Creates {@link Tab}, {@link TabModelSelector}, and {@link org.chromium.chrome.browser.tabmodel.BraveTabCreator}s in the context of a
+  * Custom Tab activity.
+  */
+ @ActivityScope
+@@ -78,13 +78,13 @@ public class CustomTabActivityTabFactory {
+         return mTabModelSelector;
+     }
+ 
+-    /** Creates a {@link ChromeTabCreator}s for the custom tab. */
```

Comment:
these methods don't have to change because BraveTabCreator is a subclass of ChromeTabCreator and you can cast if you need anything in `BraveTabCreator` that is not in `ChromeTabCreator`